addons/l10n_latam_invoice_document/wizards/account_debit_note.py

```diff
@@ -5,12 +5,16 @@ class AccountDebitNote(models.TransientModel):
 
     _inherit = 'account.debit.note'
 
-    def create_debit(self):
```

Comment:
Please do not remove methods in stable (even more if it is a public one)

Reply:
We are not deleting the method, we were only extending the method.
The proposed change only extend a different method to make it work as we expected

Best Regards